sdk/python/experimental/hello/main.py

```diff
@@ -17,8 +17,8 @@ async def html(self) -> str:
             return await (
                 client.container()
                 .from_("alpine")
-                .exec(["apk", "add", "curl"])
-                .exec(["curl", "https://dagger.io/"])
+                .withExec(["apk", "add", "curl"])
+                .withExec(["curl", "https://dagger.io/"])
```

Comment:
snake_case?

Reply:
Went crazy with replacing and did the same as TS. Fixed